docs/_static/nuts-registry.yaml

```diff
@@ -128,6 +142,28 @@ paths:
             text/plain:
               schema:
                 type: string
+  /api/organization/{id}/refresh-cert:
+    post:
+      summary: "Refreshes the organization's certificate."
+      description: "New organization certificate is issued using existing keys. If there are no keys, they're generated."
+      operationId: "refreshOrganizationCertificate"
+      tags:
+        - organizations
+      parameters:
+        - name: id
+          in: path
+          description: "URL encoded identifier"
+          required: true
+          example: "urn:oid:2.16.840.1.113883.2.4.6.1:00000007"
+          schema:
+            type: string
+      responses:
```

Comment:
No 400 code for unknown orgs (or where the private key is missing)?

Reply:
Added code 400 for unknown orgs, missing private keys should've been detected (and fixed) at startup, so that'll lead to status 500 (which is fine in my opinion).